### ISSUE TITLE
Fix the use of $COMOUT for input data in exrtofs_glo post scripts (bugzilla #1139)

### ISSUE THREAD

**Issue by DanIredell-NOAA** [NOAA-EMC/RTOFS_GLO#18]
http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=1139

Please fix the use of $COMOUT for input data in exrtofs_glo post scripts at next upgrade.

Please fix the following link to $COMOUT as input data, use $COMIN instead - 
```
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n00.archs.a archv.a
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n00.archs.b archv.b
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n00.arche.b arche.b
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n00.arche.a arche.a
scripts/exrtofs_glo_post_2.sh:    ln -s -f $COMOUT/${arfile_tplate}.a archv.a
scripts/exrtofs_glo_post_2.sh:    ln -s -f $COMOUT/${arfile_tplate}.b archv.b
scripts/exrtofs_glo_post_2.sh:    ln -s -f $COMOUT/${arefile_tplate}.a arche.a
scripts/exrtofs_glo_post_2.sh:    ln -s -f $COMOUT/${arefile_tplate}.b arche.b
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n-24.archs.a archv.a
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n-24.archs.b archv.b
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n-24.arche.a arche.a
scripts/exrtofs_glo_post_2.sh:     ln -s -f $COMOUT/${RUN}_${modID}.t${mycyc}z.n-24.arche.b arche.b
scripts/exrtofs_glo_post.sh:    ln -s -f $COMOUT/${arfile_tplate}.a archv.a
scripts/exrtofs_glo_post.sh:    ln -s -f $COMOUT/${arfile_tplate}.b archv.b
```

**Comment by DanIredell-NOAA** [comment]
Updated post scripts to link to archives from COMIN instead of COMOUT.

Also cleaned up these scripts to reduce duplicate completion messages and remove unused local variables.

**Comment by DanIredell-NOAA** [comment]
Also changed COMOUT to COMIN in the following
ush/rtofs_surface_day1-3.sh:  ln -sf  ${COMIN}/rtofs_glo_2ds_${fcast}_prog.nc ${DATA_opc}/.
ush/rtofs_surface_day4-5.sh:  ln -sf ${COMIN}/rtofs_glo_2ds_${fcast}_prog.nc ${DATA_opc}/.
ush/rtofs_surface_hcasts.sh:  ln -sf  ${COMIN}/rtofs_glo_2ds_${fcast}_prog.nc ${DATA_opc}/.

**Comment by DanIredell-NOAA** [comment]
NCO:
The issue in ush/rtofs_surface*sh scripts are fixed in the v2.4.4 and been verified.
